pkg/instancetype/instancetype.go

```diff
@@ -76,6 +80,49 @@ type InstancetypeMethods struct {
 
 var _ Methods = &InstancetypeMethods{}
 
+func (m *InstancetypeMethods) Expand(vm *virtv1.VirtualMachine, clusterConfig *virtconfig.ClusterConfig) (*virtv1.VirtualMachine, error) {
+	if vm.Spec.Instancetype == nil && vm.Spec.Preference == nil {
```

Comment:
I think OR is more inclusive

Reply:
We only want to return early if both are nil, otherwise we should always expand. I can add some unit tests showing that this is still useful, even if for example the VM only has a preference.

Reply:
Is there a scenario when one can exist without another?

Reply:
Yes.

Reply:
Could you please explain? My understanding was that both must be provided.